packages/elements/src/components/breadcrumb.ts

```diff
@@ -21,36 +24,49 @@ export class MutationTestReportBreadcrumbComponent extends LitElement {
     }
   }
 
-  public static styles = [bootstrap];
-
   public render() {
-    return html`
-      <ol class="breadcrumb rounded">
+    return html`<nav
+      class="container my-6 flex rounded-md border border-gray-200 bg-primary-100 px-5 py-3 text-gray-700 motion-safe:transition-max-width"
+      aria-label="Breadcrumb"
+    >
+      <ol class="inline-flex items-center">
         ${this.path && this.path.length > 0 ? this.renderLink(this.rootName, []) : this.renderActiveItem(this.rootName)}
         ${this.renderBreadcrumbItems()}
       </ol>
-    `;
+    </nav> `;
   }
 
   private renderBreadcrumbItems() {
     if (this.path) {
       const path = this.path;
-      return path.map((item, index) => {
-        if (index === path.length - 1) {
-          return this.renderActiveItem(item);
-        } else {
-          return this.renderLink(item, path.slice(0, index + 1));
+      return repeat(
```

Comment:
This way lit can reuse DOM elements when changing the breadcrumb. For example going from a deep path upwards (but not all the way) it can reuse all the path elements that are unchanged